reflex/components/core/layout/stack.py

```diff
@@ -0,0 +1,108 @@
+"""Stack components."""
+
+
+from typing import Literal
+
+from reflex.components.component import Component
+from reflex.components.el.elements.typography import Div
+from reflex.vars import Var
+
+
+class Stack(Div):
+    """A stack component."""
+
+    # The direction of the stack.
+    justify: Var[Literal["start", "center", "end"]]
+
+    # The alignment of the stack.
+    align: Var[Literal["start", "center", "end", "stretch"]]
+
+    # The spacing between each stack item.
+    spacing: Var[str]
```

Comment:
a bit unexpected that these are given as props, considering they wouldn't render correctly if they were set after the fact (outside of create).

i don't think anything should be a prop unless it is expected to be rendered into the tag for the frontend code... there are some exceptions throughout the codebase, but in those cases, we explicitly strip out the "fake props" inside `_render`